experimental/packages/opentelemetry-instrumentation-http/test/functionals/utils.test.ts

```diff
@@ -457,11 +299,12 @@ describe('Utility', () => {
         request,
         {
           component: 'http',
-          semconvStability: SemconvStability.OLD,
         },
         diag
       );
-      assert.strictEqual(attributes[SEMATTRS_HTTP_TARGET], '/user/?q=val');
+      const path = String(attributes[ATTR_URL_PATH] ?? '');
+      const query = String(attributes[ATTR_URL_QUERY] ?? '');
+      assert.strictEqual(path + query, '/user/?q=val');
```

Comment:
I believe that once the '?' is dropped from ATTR_URL_QUERY, this will need to change to something like:

```suggestion
      assert.strictEqual(path + '?' + query, '/user/?q=val');
```